consensus/istanbul/core/roundstate_db.go

```diff
@@ -113,6 +124,13 @@ func newRoundStateDB(path string, opts *RoundStateDBOptions) (RoundStateDB, erro
 		logger: logger,
 	}
 
+	rsdb.rsRLPMeter = metrics.NewRegisteredMeter(namespace+"rs/rlp/encoding/size", nil)
+	rsdb.rsRLPEncTimer = metrics.NewRegisteredTimer(namespace+"rs/rlp/encoding/duration", nil)
+	rsdb.rsDbSaveTimer = metrics.NewRegisteredTimer(namespace+"rs/db/save/time", nil)
+	rsdb.rcvdRLPMeter = metrics.NewRegisteredMeter(namespace+"rcvd/rlp/encoding/size", nil)
```

Comment:
I remember we used rcvd in the past for tests. I wonder if there's name for this that makes it a bit clearer for a reader. As in , the entry for the collection of received consensus messages.

Reply:
The type struct is called `rcvd`, so I thought I would just follow the convention. I also updated the comment next to the definition of the metric. Would it be enough or you want to change it here? If so, what exactly you would like to change, the name of the metric or the path or both?

Reply:
Yes, iirc I named that struct and I regret it highly xD. Maybe it's ok, I just felt that if you read the metric name, without the context of it being in this specific piece of code, it's really tough to understand what it is. That being said, it's probably the same with other metrics, so maybe it's ok.